src/atlantis/manager/client/router.go

```diff
@@ -107,6 +107,22 @@ type UpdateTrieCommand struct {
 	Reply    ManagerUpdateTrieReply
 }
 
+func (c *UpdateTrieCommand) Execute(args []string) error {
```

Comment:
Well, they were definitely removed in de7790efb5565ca6a595fa50cfe07944aa573a65...